src/components/GlobalStyles.js

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import { Global, css } from "@emotion/core";
+
+function GlobalStyles() {
+  return (
+    <Global
+      styles={theme => css`
+    *,
+    *:after,
+    *:before {
+        box-sizing: border-box;
+    }
+    
+    body {
+        margin: 0;
+        height: 100vh;
+        /* background: ${props => props.theme.images.background};
+        background-repeat: no-repeat;
+        background-attachment: fixed; */
+    }
+    
+    main {
```

Comment:
Is it possible to use this main in the app.js?

Reply:
probably yes